### PR TITLE
Hw4

### PR DESCRIPTION
**Pull request by marto55**

### CODE REVIEW

File: homework_04/13_martin_georgiev/circles_hw.py [hunk 1]
```diff
@@ -1,27 +1,43 @@
+'''
+file with circles and util for determining position and colisions
+'''
+# pylint: disable=C0103
+# pylint: disable=R0903:
```
Comment (Teodor1331) [inline homework_04/13_martin_georgiev/circles_hw.py:5]: I am not sure for what you have had C0103 (I really do not see any problems for this, you can check this, if you want: https://pycodequ.al/docs/pylint-messages/c0103-invalid-name.html). For R0903: a simple solution is to add properties or override equal methods. I just mention it, this way is very good too.

**Review by Teodor1331 — APPROVED** [review]
I think the homework has been done correct! Congratulations!

File: homework_04/13_martin_georgiev/test_circlesHW.py [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+
+def test_is_circle():
+    assert isinstance(Circle(Point(1, 1), 1), Circle)
+    assert isinstance(Circle(Point(1.456, 1.234), 0.23), Circle)
+
+    try:
+        # wrong input so this should trow error if the code works correctly
+        assert not isinstance(Circle(Point("eee", 1), 1), Circle)
+    except:
+        pass
```
[the remark this answers is not in the document]
Comment (marto55) [reply homework_04/13_martin_georgiev/test_circlesHW.py:11]: mb

[the remark this answers is not in the document]
Comment (marto55) [reply homework_04/13_martin_georgiev/test_circlesHW.py:11]: ty

File: .github/workflows/homework04_13_martin_georgiev.yml [hunk 3]
```diff
@@ -0,0 +1,38 @@
+name: Homework04 13 Martin Georgiev
```
[the remark this answers is not in the document]
Comment (valentinvarbanov) [reply .github/workflows/homework04_13_martin_georgiev.yml:1]: Homework 04 - xx Firstname Lastname

Comment (valentinvarbanov) [inline .github/workflows/homework04_13_martin_georgiev.yml:1]: file in not named correctly
homework_04_xx_firstname_lastname.yaml

**Pull request merged**